flink-libraries/flink-table/src/main/scala/org/apache/flink/table/codegen/calls/FunctionGenerator.scala

```diff
@@ -290,6 +291,15 @@ object FunctionGenerator {
     Seq(),
     new CurrentTimePointCallGen(SqlTimeTypeInfo.TIMESTAMP, local = true))
 
+  // Make ROWTIME() return the local timestamp
+  // The function has to be executable as in windowed queries it is used
+  // in the GroupBy expression. The results of the function, however, does
+  // not matter.
+  addSqlFunction(
+    EventTimeExtractor,
+    Seq(),
+    new CurrentTimePointCallGen(SqlTimeTypeInfo.TIMESTAMP, local = true))
```

Comment:
Ah, yes. You are right. It is still called in the DataStreamCalc and cannot be easily removed as you noted.

Alright, then I'd suggest to just emit a casted `null`. This is not very nice, as it might also be called at any other place but since we will remove the marker function soon, it should not be a big issue.